packages/extension-polkagate/src/hooks/useReservedDetails.ts

```
@@ -1,27 +1,26 @@
-// @ts-nocheck
-
-import type { Option } from '@polkadot/types';
+import type { Option, StorageKey } from '@polkadot/types';
 import type { Balance } from '@polkadot/types/interfaces';
-import type { AccountId } from '@polkadot/types/interfaces/runtime';
-import type { PalletMultisigMultisig, PalletPreimageRequestStatus, PalletRecoveryRecoveryConfig, PalletReferendaReferendumInfoRankedCollectiveTally, PalletReferendaReferendumStatusRankedCollectiveTally, PalletSocietyBid, PalletSocietyCandidacy } from '@polkadot/types/lookup';
-import type { BN } from '@polkadot/util';
+// @ts-ignore
+import type { PalletSocietyBid, PalletSocietyCandidacy } from '@polkadot/types/lookup';
+import type { AnyTuple, Codec } from '@polkadot/types-codec/types';
+import type { Proxy } from '../util/types';
```

Biome (1.9.4)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)